lib/workers/global/config/parse/file.ts

```diff
@@ -1,8 +1,25 @@
+import { readFileSync } from 'fs-extra';
+import { load } from 'js-yaml';
+import JSON5 from 'json5';
 import upath from 'upath';
 import { migrateConfig } from '../../../../config/migration';
-import type { AllConfig } from '../../../../config/types';
+import type { AllConfig, RenovateConfig } from '../../../../config/types';
 import { logger } from '../../../../logger';
 
+export function getParsedContent(file: string): RenovateConfig {
+  const rawContent = readFileSync(file, 'utf8');
```

Comment:
I thought you might ask me to convert this to `readFile()` as with the previous PR :) but the async creep seemed like it would require more refactoring outside the scope of this PR, so I left it as is. But I can try to go ahead with async if needed.

Reply:
Can you send a refactor PR first to make that async?

Reply:
Done in https://github.com/renovatebot/renovate/pull/12649.

Reply:
Merged, although I have a question about the switch now, will open a new thread.